python/ml4ir/applications/ranking/model/metrics/metrics_helper.py

```diff
@@ -231,6 +231,59 @@ def compute_aux_metrics_on_query_group(
     return pd.Series(aux_metrics_dict)
 
 
+def compute_groupwise_running_variance_for_metrics(metric_list, group_metric_running_variance_params, running_stats_df, group_key):
```

Comment:
@mohazahran Why can't this function be called from `get_grouped_stats()`? That way you can remove the call from `ranking_model`